Gruntfile.js

```diff
@@ -59,10 +49,10 @@ module.exports = (grunt) => {
 
     grunt.registerTask('updateLists', 'Update data lists used by the grade calculation', [
         'exec:setup',
-        'execute:entityMap',
-        'execute:tosdr',
-        'execute:polisis',
-        'execute:trackersWithParentCompany'
+        'exec:entityMap',
+        'exec:tosdr',
```

Comment:
this doesn't really work anymore - it tries to download https://tosdr.org/index/services.json which is 404

Reply:
In fact, I'm pretty sure that the whole `updateLists` action can be removed together with the 'data-scripts' folder (that haven't been touched in 4y).